Firestore/Example/FuzzTests/FuzzingResources/Serializer/Corpus/ConvertTextToBinary.sh

```diff
@@ -12,26 +14,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Converts text protos to binary protos and writes the output binary protos to
-# the output folder that is specified in SCRIPT_OUTPUT_FILE_0, which is
-# retrieved from the Output Files of the Run Script Build Phase that executes
-# this script.
+# Converts text protos to binary protos. Text protos are retrieved from the
+# folder defined by SCRIPT_INPUT_FILE_0 and the generated binary protos are
+# stored in the folder defined by SCRIPT_OUTPUT_FILE_0. Both SCRIPT_INPUT_FILE_0
+# and SCRIPT_OUTPUT_FILE_0 are defined in the Run Script Build Phase of the
+# XCode build target Firestore_FuzzTests_iOS that excecutes this script.
 
 # Directory that contains the text protos to convert to binary protos.
-text_protos_dir=${SCRIPT_INPUT_FILE_0}
+text_protos_dir="${SCRIPT_INPUT_FILE_0}"
 
 # Create a folder to write binary protos to. This is our corpus.
-binary_protos_dir=${SCRIPT_OUTPUT_FILE_0}
-mkdir -p ${binary_protos_dir}
+binary_protos_dir="${SCRIPT_OUTPUT_FILE_0}"
+mkdir -p "${binary_protos_dir}"
 
 echo "Converting text proto files in directory: $text_protos_dir"
 echo "Writing binary proto files to directory: $binary_protos_dir"
 
 # Run proto conversion command for each file content.
-for text_proto_file in $text_protos_dir/*
+for text_proto_file in "${text_protos_dir}"/*
 do
-  file_name=$(basename -- "$text_proto_file")
-  file_content=`cat $text_proto_file`
+  file_name="$(basename -- "${text_proto_file}")"
```

Comment:
I don't think quotes nest, so this is likely interpreted as
```
"$(basename -- "
${text_proto_file}
")"
```
I think you should remove the internal quotes around `${text_proto_file}`.

Reply:
Done.

Reply:
Oh, thanks, I didn't know that.

Reply:
Put them back on only inside `$()` commands